### DISCUSSION TITLE
Windows Package Manager 1.7.2722-preview

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#3717] [Releases]
This is the third development build after the Windows Package Manager 1.6 build for Windows 10 (1809+) and Windows 11. This build will be released to Windows Insider Dev builds and Windows Package Manager Insiders. 

Experimental features are enabled in this release. Run `winget features` to see which experimental features are enabled or disabled. Add the following to your settings (`winget settings`) file to enable the experimental features.

```JSON
    "experimentalFeatures": {
	  "directMSI": true
          "windowsFeature": true
    },
```

Windows Package Manager also includes [Winget configuration](https://learn.microsoft.com/windows/package-manager/configuration/), which automatically handles the setup and configuration requirements for an ideal development environment on your Windows machine. WinGet configuration file helps with installing and managing software packages, applications, programming languages, frameworks, tools, or settings necessary for a project. 

Check out our [session](https://aka.ms/BRK280H) at Microsoft Build to learn how to get your machine to a ready-to-code state.

A prerelease version of the [Microsoft.WinGet.Client PowerShell module](https://www.powershellgallery.com/packages/Microsoft.WinGet.Client/0.2.1) has been published to the PowerShell Gallery and will no longer be included as a release asset. To install the latest version of the PowerShell module, run the following command in PowerShell 7+. 

```pwsh
Install-Module -Name Microsoft.WinGet.Client
```
> The PowerShell module requires *App Installer ([winget](https://aka.ms/getwinget))* to be installed. The `Repair-WinGetPackageManager` cmdlet (work in progress) is designed to install or repair *App Installer*.

## What's Changed
* Enable COM API access to correlate with the tracking database only by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/3703
* Use correct caller name in Com startup telemetry event by @yao-msft in https://github.com/microsoft/winget-cli/pull/3711

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.7.2711-preview...v1.7.2722-preview

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.7.2722-preview'>Windows Package Manager 1.7.2722-preview</a>.</em>

**Comment by tonyping** [comment]
I'm using group policy for a couple settings. Do I have to update the policy files upon each release, or do they auto-update?

**Reply by denelon** [reply]
The policy files don't get changed with "each" release. In most cases, it's just updated localizations (if anything has been improved). The policy files do not get automatically updated. You can view the release notes to see if anything was modified or added to the policies or client behaviors as a hint to see if updating the policies would be beneficial.
